src/shared/dissect-image.c

```diff
@@ -3081,8 +3081,6 @@ int dissected_image_acquire_metadata(DissectedImage *m) {
                                 UID_INVALID,
                                 DISSECT_IMAGE_READ_ONLY|
                                 DISSECT_IMAGE_MOUNT_ROOT_ONLY|
-                                DISSECT_IMAGE_VALIDATE_OS|
-                                DISSECT_IMAGE_VALIDATE_OS_EXT|
```

Comment:
mmh in certain cases we want to fail though I think, for example when called from portabled-image-bus - maybe parameterize it, so that the dissect binary can avoid it?

Reply:
yeah, i'd agree with this, make this optional so that systemd-dissect doesn't use it, but everyone else can still use it

Reply:
Adjusted to take `extra_flags`, so that only the call from systemd-dissect is affected.